### PR TITLE
Drafts+: Fixes CSS issues on mass delete buttons

### PR DESCRIPTION
**Pull request by ThePsionic**
Also makes the CSS file itself a little easier to read.

### CODE REVIEW

**Comment by bvtsang** [comment]
I don't know if this is the fault of Drafts+ or Tweaks, but I'll list it here anyway:

1. In Tweaks, hide the Tumblr radar
2. Visit your drafts page
3. Observe that the Tumblr radar is hidden
4. Click Mass Edit Mode
5. Expect the Tumblr radar to be hidden
6. Observe that the Tumblr radar is visible

**Comment by bvtsang** [comment]
This might be a nitpick, so you're free to ignore it:

1. Visit your drafts page
2. Observe that Drafts is selected on the sidebar
3. Click Mass Edit Mode
4. Expect that Drafts remains selected
5. Observe that the Drafts menu option disappears

File: Extensions/drafts_plus.js [hunk 1]
```diff
@@ -23,9 +23,21 @@ XKit.extensions.drafts_plus = new Object({
 					'<div class="hide_overflow" style="color: rgba(255, 255, 255, 0.5) !important; font-weight: bold; padding-left: 10px; padding-top: 8px;">Mass Edit Mode</div>' +
 				'</a>' +
 			'</li>' +
-			'<li class="no_push" style="height: 36px;"><a href="#" onclick="return false;" id="xshrinkposts_button">' +
-				'<div class="hide_overflow" style="color: rgba(255, 255, 255, 0.5) !important; font-weight: bold; padding-left: 10px; padding-top: 8px;">Shrink Posts <div class="count" style="padding-top: 8px;">off</div> </div>' +
-			'</a></li>' +
+			'<li class="no_push" style="height: 36px;">' +
```
Comment (bvtsang) [inline Extensions/drafts_plus.js]: Can we move `height: 36px` into something like `#drafts_plus_sidebar > li`? It's duplicated for each menu option here.

File: Extensions/drafts_plus.js [hunk 2]
```diff
@@ -23,9 +23,21 @@ XKit.extensions.drafts_plus = new Object({
...
+				'<a href="#" onclick="return false;" id="xshrinkposts_button">' +
+					'<div class="hide_overflow" style="color: rgba(255, 255, 255, 0.5) !important; font-weight: bold; padding-left: 10px; padding-top: 8px;">Shrink Posts <div class="count" style="padding-top: 8px;">off</div></div>' +
```
Comment (bvtsang) [inline Extensions/drafts_plus.js]: Can we do something like the following?

```css
#drafts_plus_sidebar .hide_overflow {
    color: rgba(255, 255, 255, 0.5) !important;
    font-weight: bold;
    padding-left: 10px;
    padding-top: 8px;
}
```

This is currently duplicated for each menu item.

**Comment by ThePsionic** [comment]
Both of these things are out-of-scope for this PR; this is only for minor CSS fixes.

**Comment by bvtsang** [comment]
LGTM

@homu r+

**Pull request merged**